src/decision/include/subroutines/bottom_line/FollowLine.h

```diff
@@ -0,0 +1,31 @@
+//
+// Created by drdc-s2632 on 17/07/18.
+//
+
+#ifndef PROJECT_FOLLOWLINE_H
+#define PROJECT_FOLLOWLINE_H
+
+#include "Subroutine.h"
+#include <line_detect/LineDetectMsg.h>
+
+/*** Communicating Class {findLine, adjustToLine, followLine} ***/
```

Comment:
Having functions here doesn't make a lot of sense. They're likely to get  out of date really fast and are strongly defined below anyhow.

Reply:
fixed